### DISCUSSION TITLE
Strange things, garbage on screen

### DISCUSSION THREAD

**Discussion by drakonia667** [frntc/Sidekick64#13] [Q&A]
Hello,
After soldering Sidekick and powered it on and C64 too, i have blinking LEDs (first and third) with garbage on the screen. I've connected cart to pin28 of CPU (8500), processor jumper is set to 0.
Any ideas? Can this be caused by a bad microSD card? I've tried three different cards. No effect. 
![IMG_20210828_135538](https://user-images.githubusercontent.com/25977974/131221776-6a012a03-ff70-449f-9171-54a0e113642d.jpg)

**Comment by frntc** [comment]
Which display do you have? Is it's type correctly set in the .CFG and the solder jumps set? If yes, you should definitely see a logo on the screen (you can also check if the RPi displays a Sidekick-logo via HDMI). With the TFT-displays LED 1+2 should blink alternating, with the OLED LED1->4 and back.

If you don't get this, then it doesn't boot up correctly. I think it's unlikely that 3 SD cards are damaged, how did you format them?

**Reply by drakonia667** [reply]
I've got a SK logo on the display. I'm using a 240x240 TFT screen like on the photo. Jumper are soldered correctly.
I will try another card today. Cards are formatted by SD Card Formatter.
![ekran](https://user-images.githubusercontent.com/25977974/132179424-3d52d995-5092-4be6-aeb5-1bb48bab27b5.jpg)

**Comment by frntc** [comment]
Then it boots correctly, the SD card is ok. LED 1 and 2 should blink alternating (this is the sign that the Sidekick "reads" the Phi2-signal, i.e. clock frequency of the C64). Do you observe this?

The IC where the clock signal goes through is the one closest to the RPi connector (the one which is fully visible on your photo), maybe you want to check the solder connections. 

You don't have a jumper on the A13/Btn block, right? (you should choose one option, no matter which).

**Reply by drakonia667** [reply]
On the new card (Goodram, I tried SanDisk cards before) everything works fine. Time to have fun! :)
![IMG_20210906_192820](https://user-images.githubusercontent.com/25977974/132291713-e258c759-495a-4b9c-96cc-047eafbe8d33.jpg)

**Comment by frntc** [comment]
Strange that the SD card was selectively non-working, but good that it now does!
